phpstan.neon.dist

```diff
@@ -0,0 +1,32 @@
+parameters:
+    level: 1
+    paths:
+        - ./
+    excludePaths:
+        - vendor/
```

Comment:
not sure this is needed

Reply:
Currently `fonts/` must be excluded, otherwise phpstan takes too long.

Reply:
I was commenting about `vendor/` :)

Reply:
When whole `./` repo is included, yes, `vendor/` excluded is needed. Tested.